doc_source/what-is-batch.md

```diff
@@ -1,8 +1,8 @@
 # What Is AWS Batch?<a name="what-is-batch"></a>
 
-AWS Batch enables you to run batch computing workloads on the AWS Cloud\. Batch computing is a common way for developers, scientists, and engineers to access large amounts of compute resources, and AWS Batch removes the undifferentiated heavy lifting of configuring and managing the required infrastructure\. AWS Batch is similar to traditional batch computing software\. This service can efficiently provision resources in response to jobs submitted in order to eliminate capacity constraints, reduce compute costs, and deliver results quickly\.
+AWS Batch enables you to run batch computing workloads on the AWS Cloud\. Batch computing is a common way for developers, scientists, and engineers to access large amounts of compute resources, and AWS Batch removes the undifferentiated heavy lifting of configuring and managing the required infrastructure, similar to traditional batch computing software\. This service can efficiently provision resources in response to jobs submitted in order to eliminate capacity constraints, reduce compute costs, and deliver results quickly\.
 
-As a fully managed service, AWS Batch enables developers, scientists, and engineers to run batch computing workloads of any scale\. AWS Batch automatically provisions compute resources and optimizes the workload distribution based on the quantity and scale of the workloads\. With AWS Batch, there is no need to install or manage batch computing software, which allows you to focus on analyzing results and solving problems\. AWS Batch reduces operational complexities, saves time, and reduces costs, which makes it easy for developers, scientists, and engineers to run their batch jobs in the AWS Cloud\.
+As a fully managed service, AWS Batch scales to any task, automatically provisioning compute resources and optimizing workload distribution based on the quantity and scale of the workloads\. AWS Batch bypasses the installation and management of batch computing software, saving time and reducing operational complexity so you can focus on analyzing results and solving problems\.
```

Comment:
I agree that the 'scientists, developers, and engineers' phrase gets repetitive, but I'm wary of the "scales to any task" and "bypasses the installation and management..." suggestions. They imply that AWS Batch is an actor that is doing something instead of a service that enables customers to do work.

Can you focus more on just reducing the repetition that you identified, and then I can probably merge this.